amd_tressfx_vulkan/src/UtilVulkan.h

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#include <vector>
+#include <vulkan\vulkan.h>
+
+#define AMD_V_RETURN(x)                                                                  \
```

Comment:
The macro name is not good. Maybe `AMD_CHECKED_VULKAN_CALL`? I know it's much longer, but it's pretty clear what it does.